x-pack/docs/en/security/auditing/output-index.asciidoc

```diff
@@ -47,3 +47,9 @@ NOTE: Audit events are batched for indexing so there is a lag before
 events appear in the index. You can control how frequently batches of
 events are pushed to the index by setting
 `xpack.security.audit.index.flush_interval` in `elasticsearch.yml`.
+The value should adhere to the `TimeValue` datatype, the default is 1s (1 second).
```

Comment:
We have a special section of the docs that outlines time value settings, we can link to that here, and then I don't think we need an example since it's already covered in that section of the docs.